spec/spec_helper_acceptance.rb

```diff
@@ -7,9 +7,5 @@
   case fact_on(host, 'operatingsystem')
   when 'CentOS'
     host.install_package('epel-release')
-  when 'Ubuntu', 'Debian'
-    host.install_package('iproute2')
-  else
-    host.install_package('iproute')
```

Comment:
`iproute` doesn't exist on FreeBSD, so that fails to install. But I also couldn't find why this was here, and acceptance tests pass fine without?!

Reply:
Acceptance tests do not run on FreeBSD unfortunately.

Reply:
They do on my laptop :)